tests/utils/helpers.cairo

```diff
@@ -102,6 +104,15 @@ namespace TestHelpers {
         return stack_;
     }
 
+    func init_memory_with_values{range_check_ptr}(
+        memory_len: felt, memory: felt*
+    ) -> model.Memory* {
+        alloc_locals;
+        let memory_ = Memory.init();
+        Memory.store_n{memory=memory_}(memory_len, memory, 0);
+        return memory_;
```

Comment:
I tend to prefer

```
let memory = ...
with memory {
    Memory.store_n(...)
}
return memory;
```

instead of using `memory_` and `{memory=memory_}` as it doesn't "multiply" the memory words